src/util.ts

```diff
@@ -0,0 +1,38 @@
+export type Protocol = {
+  protocolName: string
+  contentType: number
+  data: Buffer
+}
+
+export type BtpData = {
+  data: {
+    protocolData: Protocol[]
+  }
+  requestId: number
+}
+
+export type Claim = {
+  signature?: string
+  amount: string
+}
+
+export type Paychan = {
+  account: string,
+  amount: string,
+  balance: string,
+  publicKey: string,
+  destination: string,
+  settleDelay: number,
+  expiration?: string,
+  cancelAfter?: string,
+  sourceTag?: number,
+  destinationTag?: number,
+  previousAffectingTransactionID: string,
+  previousAffectingTransactionLedgerVersion: number
+}
+
+export type Store = {
```

Comment:
Should this be referenced in `account.js` `store`?

Reply:
the store in account.js is actually a StoreWrapper, but I'm importing that type now